libraries/ui-library/src/components/six-select/six-select.tsx

```diff
@@ -743,7 +743,7 @@ export class SixSelect {
   }
 
   private hasSelection() {
-    return this.multiple ? this.value.length > 0 : this.value !== '';
+    return this.multiple ? this.value?.length > 0 : this.value !== '';
```

Comment:
Because `value` is typed as `@Prop({ mutable: true }) value: string | string[] = '';`, we should never need to do a check for `undefined`.

Instead of checking for a `null/undefined`  value, we should ensure it can never be `null` in the first place.

The check is actually already done [whenever the value changes](https://github.com/six-group/six-webcomponents/blob/main/libraries/ui-library/src/components/six-select/six-select.tsx#L191), but it seems to be missing when when the component is initially loaded.

Also see the info at https://stenciljs.com/docs/reactive-data#watch-decorator, that says "The @Watch() decorator does not fire when a component initially loads."

I guess we should check if other components are doing this correctly.

Reply:
Now `handleValueChange` in called in the `componentWillLoad` lifecycle method. Other components (for example `six-input` also don't do this correctly)